odoo/addons/base/models/ir_actions.py

```diff
@@ -392,7 +392,8 @@ class IrActionsActUrl(models.Model):
 
     type = fields.Char(default='ir.actions.act_url')
     url = fields.Text(string='Action URL', required=True)
-    target = fields.Selection([('new', 'New Window'), ('self', 'This Window')],
+    target = fields.Selection([('new', 'New Window'), ('self', 'This Window'),
+                               ('download', 'This Window without BlockUI')],
```

Comment:
"This Window without BlockUI" isn't really helpful here. I guess I'd go for "Download link" or simply "Download" instead